packages/lib-subject-viewers/.gitignore

```diff
@@ -0,0 +1,5 @@
+build
+dist
+
+# Ignore static asset generation
+storybook-static/*
```

Comment:
Why is this included for this library?

Reply:
I tested all the yarn scripts and `yarn build-storybook` generates the static assets in this directory.. Which I'm assuming we don't want to commit to the repository. As a safety mechanism, I added this line so that the files that get generated from that script can be preemptively ignored.